FAQ.md

```diff
@@ -123,26 +123,26 @@ Users can then choose to upgrade their components when necessary. A great
 
 ### How do I request a new component to be added to the system?
 
-The best place to start is by looking for existing conversation in [the community forums](community.service.gov.au), followed by reviewing the [Contribution Guidelines.](https://github.com/govau/uikit/blob/master/CONTRIBUTING.md)
+The best place to start is by looking for existing conversation in [the community forums](community.service.gov.au), followed by reviewing the [Contribution Guidelines.](https://github.com/govau/design-system-components/blob/master/CONTRIBUTING.md)
 
 
 
-### How do I add the UI-Kit to my build process? How do I add custom code/variables/overrides?
-Build processes can be complicated, we recommend first looking at our [UI-Kit starter pack](https://github.com/govau/uikit-starter). This is a simple
-implementation of the UI-Kit showing how to include the UI-Kit into your project. After you `npm install` and run `npm run watch` you should have some generated
+### How do I add the components to my build process? How do I add custom code/variables/overrides?
+Build processes can be complicated, we recommend first looking at our [components starter pack](https://github.com/govau/design-system-components-starter). This is a simple
+implementation of the components showing how to include the components into your project. After you `npm install` and run `npm run watch` you should have some generated
```

Comment:
implementation of the components showing how to include the design system into your project